src/components/RequestsFilters/RequestsFilters.test.js

```diff
@@ -0,0 +1,77 @@
+import '__mock__/';
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import RequestsFilters from './RequestsFilters';
+
+jest.mock('./PickupServicePointFilter', () => ({
+  PickupServicePointFilter: jest.fn(() => <div>PickupServicePointFilter</div>),
+}));
+
+jest.mock('./RequestLevelFilter', () => ({
+  RequestLevelFilter: jest.fn(() => <div>RequestLevelFilter</div>),
+}));
+
+const props = {
+  activeFilters: {
+    requestStatus: ['Open'],
+    requestType: ['Hold'],
+    pickupServicePoints: ['1'],
+    tags: ['Urgent'],
+  },
+  onChange: jest.fn(),
+  onClear: jest.fn(),
+  resources: {
+    servicePoints: { records: [{ id: '1', name: 'Service Point 1' }] },
+    tags: { records: [{ label: 'Urgent' }] },
+  },
+  titleLevelRequestsFeatureEnabled: false,
+};
+
+describe('RequestsFilters', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  beforeEach(() => {
+    render(<RequestsFilters {...props} />);
+  });
+  it('should click the onClear filters', () => {
+    userEvent.click(screen.getByTestId('clear-requestType'));
+    userEvent.click(screen.getByTestId('clear-requestStatus'));
+    userEvent.click(screen.getByTestId('clear-tags'));
+    expect(props.onClear).toHaveBeenCalled();
+  });
```

Comment:
Could you create 3 separate tests (1 for each "click" action) and check that `onClear` was called with correct argument?